src/components/VideoCard/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import { Card, CardImg, CardContent, CardTitle, CardDescription } from './styled';
+import ThemeContext from '../../state/ThemeContext';
+
+const VideoCard = ({ video }) => {
+  const { snippet, id } = video;
+  const { stateTheme } = useContext(ThemeContext);
+  const { theme } = stateTheme;
+
+  return (
+    <Link to={{ pathname: `/${id.videoId}`, state: { video } }}>
```

Comment:
If for some reason the Youtube response comes with an empty or missing id, you could get an error at `<Link>` but you can use defaults on destructuring:
```suggestion
  const { snippet, id: { videoId } = {} } = video;
  const { stateTheme } = useContext(ThemeContext);
  const { theme } = stateTheme;

  return (
    <Link to={{ pathname: `/${videoId}`, state: { video } }}>
```